src/components/interfaces.ts

```diff
@@ -8,6 +8,6 @@ export type CalciteBlockSectionToggleDisplay = "button" | "switch";
 
 export type FlowDirection = "advancing" | "retreating";
 
-export type CalciteActionAppearance = "solid" | "clear";
```

Comment:
`calcite-action.tsx` still references this: https://github.com/Esri/calcite-app-components/blob/e1af1d60ce4698a2a41b72b5237d1ce4a862d772/src/components/calcite-action/calcite-action.tsx#L3